compiler/AST/build.cpp

```diff
@@ -2102,7 +2102,13 @@ BlockStmt* convertTypesToExtern(BlockStmt* blk) {
       if (!de->init) {
         Symbol* vs = de->sym;
         PrimitiveType* pt = new PrimitiveType(NULL);
-        DefExpr* newde = new DefExpr(new TypeSymbol(vs->name, pt));
+
+        TypeSymbol* ts = new TypeSymbol(vs->name, pt);
+        if (VarSymbol* theVs = toVarSymbol(vs)) {
+          ts->doc = theVs->doc;
+        }
```

Comment:
I was wondering why we didn't copy the flags or anything from the symbol wholesale, but upon thinking about it more, it made sense to me why we wouldn't.  Since the ability to document something is more general of a property and something we want in both extern and Chapel cases, I think it's fair to only copy the NO_DOC flag here and not do anything further.  @bradcray, as the person who initially added this function and capability, does that sound like a reasonable assessment of the current situation, or do you think there are other flags we should carry over (in a separate pull request, probably)?